tofhir-server/src/main/scala/io/tofhir/server/endpoint/MappingContextEndpoint.scala

```diff
@@ -81,6 +88,67 @@ class MappingContextEndpoint(mappingContextRepository: IMappingContextRepository
     }
   }
 
+  /**
+   * Route to update the mapping context header
+   * @param projectId project id
+   * @param id mapping context id
+   * @return
+   */
+  private def updateMappingContextHeader(projectId: String, id: String): Route = {
```

Comment:
Can you please follow the same convention while naming the functions inside an Endpoint class?